src/app/containers/PageContainer/components/DefaultHeaderComponent/DefaultHeaderComponent.tsx

```diff
@@ -464,6 +469,16 @@ export const DefaultHeaderComponent: React.FC = () => {
                 >
                   {t(translations.mainMenu.wallet)}
                 </NavLink>
+                {currentNetwork === AppMode.MAINNET && (
+                  <a
+                    className="tw-header-link tw-mr-2 2xl:tw-mr-3"
+                    href={zeroUrl}
+                    data-action-id="header-zero-link"
+                  >
+                    {t(translations.mainMenu.zero)}
+                  </a>
+                )}
+
                 {currentNetwork === AppMode.TESTNET && (
```

Comment:
This is not correct because it hides the whole `Labs` section on mainnet but you only want to hide `Perpetuals`. Take a look at https://github.com/DistributedCollective/Sovryn-frontend/pull/2207 , it's solved there.

Also, you didn't add `Zero` to the mobile menu.